synapse/rest/client/read_marker.py

```diff
@@ -40,9 +40,11 @@ def __init__(self, hs: "HomeServer"):
         self.read_marker_handler = hs.get_read_marker_handler()
         self.presence_handler = hs.get_presence_handler()
 
-        self._known_receipt_types = {ReceiptTypes.READ, ReceiptTypes.FULLY_READ}
-        if hs.config.experimental.msc2285_enabled:
-            self._known_receipt_types.add(ReceiptTypes.READ_PRIVATE)
+        self._known_receipt_types = {
+            ReceiptTypes.READ,
+            ReceiptTypes.FULLY_READ,
+            ReceiptTypes.READ_PRIVATE,
+        }
```

Comment:
I think we technically don't need to since they won't cause any problems, afaict. Though we could if we want to free up the database though I think I'd prefer to do that separately?

Reply:
> I think we technically don't need to since they won't cause any problems, afaict?

Mostly I'm concerned that we would start leaking them out if we don't have specific code to handle them!

Reply:
True! I suppose it's not a big issue since this was previously considered unstable though it might be worth the time - I'll try to have a look into it. I think I only need to edit `filter_out_private_receipts()`?

Reply:
Yeah, I think keeping the unstable prefix as a constant and keeping the filtering in `filter_out_private_receipts` likely makes sense for now. In the future we could do a background update to remove them and then remove the filtering?

This should give clients a bit of transition time, but also not break things for users. I hope this wouldn't cause folks receipts to jump backwards though...